script-javascript/pom.xml

```diff
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>org.apache.servicemix.bundles</groupId>
             <artifactId>org.apache.servicemix.bundles.rhino</artifactId>
-            <version>1.7R4_1</version>
+            <version>1.7.13_1</version>
```

Comment:
Do we still need ServiceMix version of the dependency? I have checked RhinoJS's 1.7.13 JAR file and it contains OSGi manifest declarations.

Reply:
We don't. 
I have replaced the servicemix dependency with non-servicemix Rhino